server/emails/templates/CommentCreatedEmail.tsx

```diff
@@ -119,6 +120,15 @@ export default class CommentCreatedEmail extends BaseEmail<
     return actorName;
   }
 
+  protected replyTo({ notification }: Props) {
+    if (notification?.user && notification.actor?.email) {
+      if (can(notification.user, "readEmail", notification.actor)) {
```

Comment:
"readEmail" permission does not allow viewers - is it intentional?
https://github.com/outline/outline/blob/7bdae0cbda21d01a431b42a3dc43344e662c1c8a/server/policies/user.ts#L41-L48

Reply:
It is… but also, it might not be neccessary to restrict to just members. A bit undecided